pages/dao/[symbol]/proposal/components/instructions/Raydium/RemoveLiquidityFromPool.tsx

```diff
@@ -61,10 +53,10 @@ const RemoveLiquidityFromPool = ({
   } | null>(null)
 
   useEffect(() => {
-    ;(async () => {
+    async function fetchLpMintInfo() {
       try {
         const { maxBalance, decimals } = await fetchLiquidityPoolData({
-          governanceKey: form.governedAccount?.governance.pubkey,
+          governanceKey: governanceAccount?.governance.pubkey,
```

Comment:
Should we ever fetch this data if `governanceAccount...` is undefined / null ?

If not, then extract `const govAccountPubkey = ...` earlier and use that as hook dependency, then just do nothing in the effect if it's falsy?